dags/top-public-synapse-projects-from-snowflake.py

```diff
@@ -28,15 +21,24 @@
 from airflow.providers.snowflake.hooks.snowflake import SnowflakeHook
 from orca.services.synapse import SynapseHook
 from slack_sdk import WebClient
+import json
 
 dag_params = {
-    "snowflake_developer_service_conn": Param("SNOWFLAKE_DEVELOPER_SERVICE_RAW_CONN", type="string"),
+    "snowflake_developer_service_conn": Param(
+        "SNOWFLAKE_DEVELOPER_SERVICE_RAW_CONN", type="string"),
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
-    # hours_time_delta is the number of hours to subtract from the current date to get the date for the query
+    # hours_time_delta is the number of hours to subtract from the current date to get
+    # the date for the query
     "hours_time_delta": Param("24", type="string"),
     "backfill": Param(False, type="boolean"),
     # backfill_date string format: YYYY-MM-DD
     "backfill_date": Param("1900-01-01", type="string"),
+    # fileview_groups is a JSON string containing an array of objects with file_view_id
+    # and group_name properties
+    # Example: '[{"file_view_id": "123456", "group_name": "Group A"},
+    #           {"file_view_id": "789012", "group_name": "Group B"}]'
+    "fileview_groups": Param(
+        '[{"file_view_id": "20446927", "group_name": "HTAN1"}]', type="string"),
```

Comment:
This is excellent, we can advertise this in #synapse channel once this is up if people wanted to add more fileviews.  There are programs like NF, AMP-AIM, and etc that would probably like this functionality added, but would be nice to show it first.

Reply:
Some of the views I'll take note of, (need feedback before we add it). NF-OSI could be a pretty safe one to add because they've asked for it in the past

* AMP-AIM : syn64332440
* ELITE: syn51753858
* HTAN2: syn63714258
* MC2: syn27210848
* NF-OSI: syn16858331
* VEOIBD: syn51397378

Reply:
Should we reach out today and get these added @thomasyu888?

Reply:
I'll reach out @BryanFauble !